### ISSUE TITLE
Options are not read by screen reader

### ISSUE THREAD

**Issue by venlahy** [jfangrad/react-dropdown-aria#38]
The dropdown options in the examples in the page https://jfangrad.github.io/react-dropdown-aria/ are not read at all by the NVDA screen reader. When I open for example the Basic Usage dropdown, and move through the options, the screen reader just says "list item 1 of 8", and "list item, not selected, 2 of 8", although it should say something like "Java, list item 1 of 8" and "JavaScript, list item, not selected, 2 of 8".

The screen reader is also silent when the dropdown is opened or closed, although it should say something in these situations, too.

**Comment by jfangrad** [comment]
Thanks for the heads up! I believe this should be fixed in v2.0.6? Let me know if you still notice any issues.

Not exactly sure what you're expecting it to say when opening/closing. Feel free to look at the updated version and maybe give an example of what you would expect there?

**Comment by venlahy** [comment]
Yes, the reading of the options was fixed in v2.0.6.

If the dropdown is given the props ariaLabel="React Simple Dropdown", and if for example the "Swift" option is selected (in the Basic Usage example in the page https://jfangrad.github.io/react-dropdown-aria/), the screen reader says now "list, Java, not selected 1 / 8" when the dropdown is focused, and nothing, when the dropdown is closed. I think that the screen reader should say something like "React Simple Dropdown, Swift, menu button" when the dropdown is focused or closed.

When the dropdown is opened, the screen reader does not say anything currently usually. I think it should say something like "open, React Simple Dropdown, menu, Swift, 3 / 8".
